src/Copy/Text.elm

```diff
@@ -755,6 +755,9 @@ t key =
         GetHelpSection1Title ->
             "Support from survivors"
 
+        GetHelpSection1Icon ->
```

Comment:
I thought we decided we didn't think these icons worked with the rest if the design?